app/clients/cloudwatch/aws_cloudwatch.py

```diff
@@ -0,0 +1,93 @@
+import json
+import os
+import re
+import time
+
+from boto3 import client
+
+from app.clients.cloudwatch import CloudwatchClient
+from app.cloudfoundry_config import cloud_config
+
+
+class AwsCloudwatchClient(CloudwatchClient):
+    """
+    This client is responsible for retrieving sms delivery receipts from cloudwatch.
+    """
+
+    def init_app(self, current_app, *args, **kwargs):
+        self._client = client(
+            "logs",
+            region_name=cloud_config.sns_region,
+            aws_access_key_id=cloud_config.sns_access_key,
+            aws_secret_access_key=cloud_config.sns_secret_key
+        )
+        super(CloudwatchClient, self).__init__(*args, **kwargs)
+        self.current_app = current_app
+        self._valid_sender_regex = re.compile(r"^\+?\d{5,14}$")
```

Comment:
[suggestion] I think this isn't being used anymore, can it be deleted?

Reply:
will do

Reply:
This still looks like it's hanging around

Reply:
It must be be something with Github not updating the PR (?).  I just synced down this branch and now the code is doing super(Client).

Reply:
I see the update now 👍🏻